src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +509,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
```

Comment:
@Sami32 I see when I check the code that the field isn't used quite the way I thought. It's quite a bad design as far as I can see, so I'm sure there are places in the code that "forgets" to set the id - so I agree.